external-wallets/wagmi-wallets/src/config.ts

```diff
@@ -0,0 +1,7 @@
+import { createConfig } from 'wagmi'
+import { base, mainnet } from 'wagmi/chains';
+
+// @ts-ignore
```

Comment:
Is it possible for us to eliminate this?

Reply:
Based on https://wagmi.sh/react/typescript it needed the typescript version 5.